ingest/stream_records.go

```diff
@@ -44,13 +46,20 @@ func (c *Client) StreamRecords(mappingID string, records []*ingestpb.Record) (
 		if err != nil {
 			return nil, errors.FromError(err)
 		}
+		resp, err := stream.Recv()
```

Comment:
How this will work, when for some reason server will not send response? It will block sending too?

Reply:
I believe this should be a separate thread. Like separate go routine which will collect everything. I think on the same we agreed when Martin asked the same question about Python SDK.

But I'm approving this, as after our discussion about Bi-Directional streams this will most likely change as well.